src/showcases/components/icon/iconAnimation.component.tsx

```diff
@@ -8,6 +8,7 @@ export const IconAnimationShowcase = () => {
   const pulseIconRef = React.useRef();
   const shakeIconRef = React.useRef();
   const infiniteAnimationIconRef = React.useRef();
+  const noAnimationIconRef = React.useRef();
```

Comment:
Thanks for taking care on the docs!
I guess there is no need to have a documented example on this if it is default behavior 😄